### ISSUE TITLE
[LIB REQUEST]: Is there any way C++ rapidjson library can be added?

### ISSUE THREAD

**Issue by HazyMrf** [compiler-explorer/compiler-explorer#6942]
### Library name

RapidJson

### Library description

Library for parsing JSON

### Library repository

https://github.com/Tencent/rapidjson

### Motivation

It is second popular after nlohmann and yet not on godbolt

**Comment by mattgodbolt** [comment]
Yes: it can be added. For those wanting to look at this: https://github.com/compiler-explorer/compiler-explorer/blob/main/docs/AddingALibrary.md

**Comment by arong** [comment]
@HazyMrf RapidJSON is now available, see [this example](https://godbolt.org/z/M3Tonrd45)
